COMMANDS.md

```diff
@@ -24,6 +25,51 @@ The most common subcommands are:
 * [`pyenv init`](#pyenv-init)
 * [`pyenv completions`](#pyenv-completions)
 
+## `pyenv help`
+
+List all available pyenv commands along with a brief description of what they do. Run `pyenv help <command>` for information on a specific command.
+
+```
+$ pyenv help
+Usage: pyenv <command> [<args>]
+
+Some useful pyenv commands are:
+   activate    Activate virtual environment
+   commands    List all available pyenv commands
+   deactivate   Deactivate virtual environment
+   doctor      Verify pyenv installation and development tools to build pythons.
+   exec        Run an executable with the selected Python version
+   global      Set or show the global Python version(s)
+   help        Display help for a command
+   hooks       List hook scripts for a given pyenv command
+   init        Configure the shell environment for pyenv
+   install     Install a Python version using python-build
+   local       Set or show the local application-specific Python version(s)
+   prefix      Display prefix for a Python version
+   rehash      Rehash pyenv shims (run this after installing executables)
+   root        Display the root directory where versions and shims are kept
+   shell       Set or show the shell-specific Python version
+   shims       List existing pyenv shims
```

Comment:
Is there any use duplicating this list here?

Reply:
I am not sure. Should I remove it? Any suggestions?

Reply:
I think just the description text that's above and below gives enough information. we already have the list of subcommands, right in this document!

Reply:
Thanks for the suggestions, updated the PR.